src/PSAppDeployToolkit/Public/Set-ADTIniSection.ps1

```diff
@@ -0,0 +1,151 @@
+        Set-ADTIniSection -FilePath "$env:ProgramFilesX86\IBM\Notes\notes.ini" -Section 'Notes' -Content @{}
+
+       Sets the 'Notes' section to be empty by sending an empty hashtable.
+
+    .EXAMPLE
+        Set-ADTIniSection -FilePath "$env:ProgramFilesX86\IBM\Notes\notes.ini" -Section 'Notes' -Content $null
+
+       Sets the 'Notes' section to be empty by sending a null value.
+
+    .NOTES
+        An active ADT session is NOT required to use this function.
+
+        Tags: psadt<br />
+        Website: https://psappdeploytoolkit.com<br />
+        Copyright: (C) 2025 PSAppDeployToolkit Team (Sean Lillis, Dan Cunningham, Muhammad Mashwani, Mitch Richters, Dan Gough).<br />
+        License: https://opensource.org/license/lgpl-3-0
+
+    .LINK
+        https://psappdeploytoolkit.com/docs/reference/functions/Set-ADTIniSection
+    #>
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$FilePath,
+
+        [Parameter(Mandatory = $true)]
+        [ValidateScript({
+                if ([string]::IsNullOrWhiteSpace($_))
+                {
+                    $PSCmdlet.ThrowTerminatingError((New-ADTValidateScriptErrorRecord -ParameterName Path -ProvidedValue $_ -ExceptionMessage 'The specified section cannot be null, empty, or whitespace.'))
+                }
+                return ![System.String]::IsNullOrWhiteSpace($_)
+            })]
+        [string]$Section,
+
+        [Parameter(Mandatory = $true)]
+        [AllowNull()]
+        [System.Collections.IDictionary]$Content,
+
+        [Parameter(Mandatory = $false)]
+        [System.Management.Automation.SwitchParameter]$Force
+    )
+
+    begin
+    {
+        Initialize-ADTFunction -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+    }
+
+    process
+    {
+        try
+        {
+            try
+            {
+                # Create the INI file if it does not exist.
+                if (![System.IO.File]::Exists($FilePath))
+                {
+                    if (!$Force)
+                    {
+                        $naerParams = @{
+                            Exception = [System.IO.FileNotFoundException]::new("The file [$FilePath] is invalid or was unable to be found.")
+                            Category = [System.Management.Automation.ErrorCategory]::ObjectNotFound
+                            ErrorId = 'FilePathNotFound'
+                            TargetObject = $FilePath
+                            RecommendedAction = "Please confirm the path of the specified file and try again, or add -Force to create a new file."
+                        }
+                        throw (New-ADTErrorRecord @naerParams)
+                    }
+                    Write-ADTLogEntry -Message "Creating INI file: $FilePath."
+                    $null = New-Item -Path $FilePath -ItemType File -Force
+                }
+
+                if ($null -eq $Content -or $Content.Count -eq 0)
```

Comment:
As mentioned above in another area of the review, I'd put the most likely scenario in the if statement, which in this case to me would be a populated dictionary.